### ISSUE TITLE
Not getting the most recent blueprints?

### ISSUE THREAD

**Issue by Ranadiel** [factubsio/BubblePrints#10]
So something I have noticed is that I am not getting the most recent blueprints when using BubblePrints (just checked in version 0.23.2, but I have noticed it in earlier versions too). The current version of Wrath is 1.2.2a, but the selectable versions in BubblePrints go from 1.2.0f_1 to 1.2.0A_1 (a version number never used) to 1.2.0A_2 (also a version number never used).

Not sure if they changed the way versioning works in 1.2.1, but checking for files that I know were introduced in 1.2.1 (e.g., the Horns of Naraga item), I am not seeing them in BubblePrints even when I use names (e.g., Artifact_HornsOfNaragaItem) or GUIDs (e.g., 96acb4453b384288815a71a07413e453) for them from Data Viewer. Similarly blueprints that I know were changed in 1.2.1 (e.g., answer list GUID 5513a41eae994b599b6a5daa9cf0109f) are showing their pre-1.2.1 state (in the case of 5513a41eae994b599b6a5daa9cf0109f, it has 3 answers instead of 5).

Entirely possible this could be user error, but I'm not sure what I could be doing wrong because I have "check for new blueprints" to true.

**Comment by factubsio** [comment]
I need to import new version but I am a lazy bubble. I shall do this weekend!

**Comment by factubsio** [comment]
The weird capital A versions were because stupid owl didn't bump the actual version number from beta and used a double letter suffix which confused me.

The last number is a bubble version so I can add new features to the file but keep the game version the same.

**Comment by factubsio** [comment]
Ps I didn't get round to doing this last weekend but will do on Friday this week.

**Comment by Ranadiel** [comment]
No worries on missing it. Fyi, 1.3 is coming out on the 21st, so if it is a time consuming process might want to wait another week so you don't have to do it twice in a short period.

**Comment by factubsio** [comment]
https://github.com/factubsio/BubblePrintsData/releases/tag/1.3.0i_0

Should auto-update, there is also a new version of bubbleprints that fixes a bunch of bugs!

**Comment by Ranadiel** [comment]
I don't think the problem is fixed. I updated to the latest version. I noticed the 1.3 update being downloaded. I have 1.3.0i_0 selected in bubbleprints... but I still am unable to find the blueprint for the Horns of Naraga. And I've checked some other blueprints, and they are likewise old versions that are not consistent with what Data Viewer is showing me in game.

Looking at the biz file you linked me to, Notepad++ isn't finding the GUID for Horns of Naraga (96acb4453b384288815a71a07413e453). Maybe you uploaded the wrong file as 1.3?
